docs/index.rst

```diff
@@ -292,6 +301,58 @@ Misc Options
     a colored output is provided by default when the ``termcolor`` package is
     installed.  This can be disabled by specifying this option.
 
+Tokens configuration
+````````````````````
+
+Tokens supported by Testimony can be configured with ``--tokens``,
+``--minimum-tokens`` and ``--config`` options.
+
+``--tokens`` takes comma-separated list of supported tokens. When testimony
+encounters token outside of this range, it will report it as error.
+
+``--minimum-token`` takes comma-separated list of required tokens. When
+testimony encounters test without all of tokens in this group, it will
+report it as error. Tokens specified here are automatically added to
+list of supported tokens (there is no need to specify single token
+in both ``--minimum-tokens`` and ``--tokens``).
+
+``--config`` is path to YAML configuration file. YAML file should contain
+single map (equivalent of Python dict), where keys are names of tokens
+and values are maps consisting of ``mandatory``, ``type`` and other,
+type-dependant keys. Sample config files are provided in ``tests``
+directory, as well as printed below:
+
+.. code-block:: console
```

Comment:
You mean something like `example-config.yaml` in root space of testimony, or maybe `docs`?

We don't actually build docs for testimony, but I would keep it there for now. It's probably the most discoverable place at the moment. This will become problem as we gain support for more token types, but I would just deal with it then.

Reply:
Sure, just mentioned to have that in mind